app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ConversationsController < ApplicationController
+
+  def show
+    @message = Message.new
+    @conversation = Conversation.find(params[:id])
+    @messages = @conversation.messages.order(:created_at)
```

Comment:
You could set a default scope on your association that does this by default:
```ruby
has_many :messages -> { order(:created_at) }
```
...a small, but useful, refactoring from your controller to your model...